examples/vue.test.js

```diff
@@ -2600,6 +2600,63 @@ describe('Vue', function() {
       // acquit:ignore:end
     });
   });
+  it('vue-websocket', async function() {
+    // acquit:ignore:start
+    this.timeout(10000);
+    const browser = await puppeteer.launch({ headless: false });
+    const page = await browser.newPage();
+    // acquit:ignore:end
```

Comment:
Move the `// acquit:ignore:end` to after the `function() ` line below. The `function() {` line below is just boilerplate to make it easy to transfer this code to puppeteer.